### ISSUE TITLE
More informative organization of atlases on explore page

### ISSUE THREAD

**Issue by elysekozlowski** [ncihtan/htan-portal#161]
This is a future enhancement - we should add some organization to the atlases on the main explore page maybe similar to the figure from the marker page (by disease type, organ type, etc.). Not all of the atlas names are informative to the user.

![image](https://user-images.githubusercontent.com/67977813/111648786-1c28fa80-87da-11eb-9e44-db642ed10c25.png)

**Comment by inodb** [comment]
Yeah I like this idea - Im not 100% sure if the explore page is the best location, maybe this should go on the homepage and you can click in this image/visualization to directly jump to particular filtering options in the explore page. It would also be nice to have a separate assay focused type visualization or incorporate that into this one somehow
